### PR TITLE
Complete the Badges

### PR DESCRIPTION
**Pull request by KBhautik6**
- [x] Checking the code is duplicate or not?
- [x] Is these code has bug 
- [x] Is code is check by shubhum sir or ankit sir or tara ma'am?
- [x] Are you going to merge this or not?

### CODE REVIEW

File: src/stories/Badges/Badges.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import {
+  StyledImageContainer,
+  StyledAvatarImage,
+  StyledBadgeIcon,
+  StyledImager,
+  StyledIconImageBadge,
+  StyledBadgesIcon,
+} from "./Badges.styled";
+
+export const Badges = ({ src, size, varaint, colortype, text }) => {
```
Comment (tarakhant1297) [inline src/stories/Badges/Badges.js]: @KBhautik6 re to name colortype color

File: src/stories/Badges/Badges.styled.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  ${({ size }) => css`
+    height: ${size}rem;
+    border-radius: 50%;
+  `}
+`;
+
+export const StyledBadgeIcon = styled.div`
+  position: absolute;
+  left: 52%;
+  /* border-color: ${theme?.white}; */
```
Comment (tarakhant1297) [inline src/stories/Badges/Badges.styled.js]: @KBhautik6 remove commented code

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]

File: src/stories/Badges/Badges.styled.js (hunk 2 above)
Comment (KBhautik6) [reply src/stories/Badges/Badges.styled.js]: okay

File: src/stories/Badges/Badges.js (hunk 1 above)
Comment (KBhautik6) [reply src/stories/Badges/Badges.js]: Okay i will do color

File: src/stories/Badges/Badges.styled.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  top: 66%;
+  transform: translateX(50%);
+  width: 12px;
+  height: 12px;
+  background-color: ${({ color }) => theme?.[color]};
+  border-radius: 50%;
+  display: inline-block;
+  border: 2px solid;
```
Comment (ankitadrixus08) [inline src/stories/Badges/Badges.styled.js]: use color from theme @KBhautik6

File: src/stories/Badges/Badges.styled.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  border-color: white;
```
Comment (ankitadrixus08) [inline src/stories/Badges/Badges.styled.js]: use shorthand property @KBhautik6

File: src/stories/Badges/Badges.styled.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+`;
+
+export const StyledBadgesIcon = styled(StyledBadgeIcon)`
+  background-color: ${({ color }) => theme?.[color]};
+  left: 62%;
+  top: 7%;
+  width: 15px;
+  height: 15px;
+  border-radius: 50%;
+  color: white;
```
Comment (ankitadrixus08) [inline src/stories/Badges/Badges.styled.js]: use color from theme @KBhautik6

File: src/stories/Badges/Badges.styled.js (hunk 3 above)
Comment (KBhautik6) [reply src/stories/Badges/Badges.styled.js]: okay sir

File: src/stories/Badges/Badges.styled.js (hunk 4 above)
Comment (KBhautik6) [reply src/stories/Badges/Badges.styled.js]: okay sir

File: src/stories/Badges/Badges.styled.js (hunk 5 above)
Comment (KBhautik6) [reply src/stories/Badges/Badges.styled.js]: okay sir

**Review by tarakhant1297 — APPROVED** [review]

**Review by ankitadrixus08 — APPROVED** [review]

**Pull request merged**